shell.nix

```diff
@@ -0,0 +1,8 @@
+{ pkgs ? import (fetchTarball "https://github.com/NixOS/nixpkgs/archive/b77bbeca8ba3069eea82a3711242cdc2240cc7c9.tar.gz") {} }:
+
+pkgs.mkShell {
+  buildInputs = [
+    pkgs.ruby_3_1
```

Comment:
CI is already covering 3.2, this just affect for local env. (Nix official repo does not contain 3.2 yet)